build-logic/src/main/kotlin/publish-library-convention.gradle.kts

```diff
@@ -0,0 +1,50 @@
+@file:Suppress("UnstableApiUsage")
+
+plugins {
+    id("com.vanniktech.maven.publish")
+}
+
+val versionSuffix = when (System.getenv("SNAPSHOT")) {
+    null -> "-TEST"
+    "true" -> "-SNAPSHOT"
+    else -> ""
+}
+project.version = project.version.toString() + versionSuffix
+
+mavenPublishing {
+    publishToMavenCentral(com.vanniktech.maven.publish.SonatypeHost.S01)
+
+    coordinates(
+        groupId = project.group.toString(),
+        artifactId = project.ext["artifactId"].toString(),
+        version = project.version.toString()
+    )
+
+    pom {
+        name.set("humans-mvi")
+        description.set("Simple and concise implementation of Redux/MVI approach by Humans.")
+        inceptionYear.set("2023")
```

Comment:
[Improvement] Hardcoded year. This will break in 2024 and will require manual update. Remove or https://stackoverflow.com/questions/136419/get-integer-value-of-the-current-year-in-java

Reply:
I'm not sure that you've got it right.
according to the documentation:
```
The year of the inception for the publication represented by this POM
```

so, we don't need to change this property next year.